src/plugins/discover_legacy/public/application/angular/directives/histogram.tsx

```diff
@@ -298,10 +297,10 @@ export class DiscoverHistogram extends Component<DiscoverHistogramProps, Discove
     }
 
     const rectAnnotationStyle = {
-      stroke: isDarkMode ? darkEuiTheme.euiColorLightShade : lightEuiTheme.euiColorDarkShade,
```

Comment:
@ananzh does this change also need to go into the migrated discover histogram?

Reply:
We haven't implement anything for dark mode for discover 2.0. Let me open an issue on it. 
Why do we remove this line for discover_legacy?